config/clickhouse/loc_config.xml

```diff
@@ -1,3 +1,3 @@
 <yandex>
-    <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+    <max_server_memory_usage_to_ram_ratio>0.3</max_server_memory_usage_to_ram_ratio>
```

Comment:
Unfortunately, the arm64 image does *not* read the value from memory.

Reply:
Please also update this in https://github.com/getsentry/sentry/blob/master/config/clickhouse/dist_config.xml#L2